bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -127,18 +127,8 @@ public void loadConf(URL confURL) throws ConfigurationException {
      * @param baseConf
      *          Other Configuration
      */
-    public void loadConf(AbstractConfiguration baseConf) {
-        addConfiguration(baseConf); 
-    }
-
-    /**
-     * Load configuration from other configuration object
-     *
-     * @param otherConf
-     *          Other configuration object
-     */
-    public void loadConf(Configuration otherConf) {
-        addConfiguration(otherConf);
+    public void loadConf(CompositeConfiguration baseConf) {
```

Comment:
As descripted above, configuration is an interface provided in common-configs, and not extends cloneable interface.  
This part of change is mainly to delete:
```
public void loadConf(Configuration otherConf) {
        addConfiguration(otherConf);
}
```
And change other place and compromise this method is calling.